graphsdk/src/main/java/com/microsoft/graph/model/Date.java

```diff
@@ -0,0 +1,90 @@
+package com.microsoft.graph.model;
+
+
+import java.util.Locale;
+
+/**
+ * A timezone-nonspecific date
+ */
+public class Date {
```

Comment:
Date is a little bit of an overloaded term, lets change this to `DateOnly` to make it crystal clear this is something different than the DateTime family of objects